<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>446</width>
      <height>318</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="7">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>请求序列</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>当前磁臂位置</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="location">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>SCAN</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>生成序列数量</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>生成序列范围</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="11">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="4">
      <widget class="QPushButton" name="step_bu">
       <property name="text">
        <string>单步执行</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4" colspan="2">
      <widget class="QLineEdit" name="sequences">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="6" rowspan="12">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="5">
      <widget class="QPushButton" name="execute">
       <property name="text">
        <string>全部执行</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="sequence_range"/>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="radioButton_1">
       <property name="text">
        <string>SSTF</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="6">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="4" rowspan="7" colspan="2">
      <widget class="QTextBrowser" name="execute_detail">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="11">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="sequence_num"/>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="product_seq">
       <property name="text">
        <string>点我生成</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" rowspan="11">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>product_seq</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>product()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>step_bu</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>single_step()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>execute</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>product()</slot>
  <slot>single_step()</slot>
  <slot>action()</slot>
 </slots>
</ui>
